boards/xtensa/esp32/esp32.yaml

```diff
@@ -5,6 +5,7 @@ arch: xtensa
 toolchain:
   - zephyr
 supported:
+  - adc
```

Comment:
Was this info added in all ESP32 boards related in this PR as such as ESP32-S2 and ESP32-C3?